monocle/tests/unit/common.py

```diff
@@ -36,8 +38,9 @@ def load_change(name: str) -> Tuple[Dict, List[Dict]]:
     return input_pr, xtrd_ref
 
 
-def index_dataset(eldb, name):
-    data = load_dataset(name)
+def index_dataset(eldb, name) -> None:
+    _data: List[Dict] = load_dataset(name)
+    data = list(map(lambda x: dict_to_change_or_event(x), _data))
```

Comment:
It would be easier to read with a list comprehension syntax like

data = [dict_to_change_or_event(x) for x in _data]